Add `ExactMatch` type (#237)
sindresorhus/type-fest: readme.md

- [`Entries`](source/entries.d.ts) - Create a type that represents the type of the entries of a collection.
- [`SetReturnType`](source/set-return-type.d.ts) - Create a function type with a return type of your choice and the same parameters as the given function type.
- [`Asyncify`](source/asyncify.d.ts) - Create an async version of the given function type.

> - [`ExactMatch`](source/exact-match.d.ts) - Returns a `boolean` type to determine if the two types match exactly.

Reviewer: ```suggestion
- [`ExactMatch`](source/exact-match.d.ts) - Returns a boolean for whether the given two types match exactly.
```
